docs/source/started.rst

```diff
@@ -73,7 +73,7 @@ Luckily, we offer our own `SequentialState <https://norse.github.io/norse/auto_a
 
     model = SequentialState(
         torch.nn.Linear(10, 5),
-        norse.LIFFeedForwardCell(),
+        norse.LIF(),
```

Comment:
Is it LIFCell or LIF? See above.

Reply:
I updated the documentation (mainly getting started and working with pages). It's actually easier to explain, which I guess is a good sign.

Reply:
Yeah I think that is a good sign!